layouts/states/single.html

```diff
@@ -0,0 +1,18 @@
+{{ partial "head.html" . }}
+
+  {{ $registration_type := .Params.registration_type }}
```

Comment:
This and setting `registration_template`'s value feels like something that might be defined one level up from the template, but looking at Hugo's docs, I can't see a way to define custom template variables outside of the template. Do you know if there is a way?
Even if there is, this is certainly a fine solution.